opencog/cogserver/scm/config.scm

```diff
@@ -44,6 +44,7 @@
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/sureal"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/lg-dict"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/planning"))
+(add-to-ltdl-path (string-append (getcwd) "/opencog/scm/opencog/learning"))
```

Comment:
This adds to LTDL_LIBRARY_PATH when developing without installing. I guess since most of the hr development requires installing this might not be necessary.